app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -452,6 +452,10 @@ class PatientRepository @Inject constructor(
         }
   }
 
+  fun latestPhoneNumberForPatient(patientUuid: UUID): Optional<PatientPhoneNumber> {
+    return database.phoneNumberDao().latestPhoneNumber(patientUuid).toOptional()
+  }
```

Comment:
Since we will be doing this for a lot more repository methods now, can we start marking the reactive versions of these methods as `Deprecated` yet?

Reply:
I do not specifically want to mark them as `Deprecated` because there might be use-cases where we actually want them to be reactive streams. What I am thinking is that we should prefer the non-Rx versions by default but then use the Rx versions if we really need to.

Does this make sense?

Reply:
Things like these end up becoming tribal knowledge. I wanted to avoid that. 
But it does make sense to keep the Rx versions for now in case we need them.